apps/supernova/src/AppContent.jsx

```diff
@@ -70,19 +70,33 @@ const AppContent = (props) => {
       })
     }
 
-    addMessage({
-      variant: "error",
-      text: parseError(alertsError),
-    })
+    if (parseError(alertsError) === "Failed to fetch") {
+      addMessage({
+        variant: "error",
+        text: "Sorry, there was an issue fetching the data. Possible causes could include network issues, incorrect URL, or server outages. ",
```

Comment:
Would you add the message `"Sorry, there was an issue fetching the data. Possible causes could include network issues, incorrect URL, or server outages.` to the parseError method so it is not duplicated in the code?